.prettierrc.yaml

```diff
@@ -0,0 +1,3 @@
+printWidth: 100
+tabWidth: 2
+singleQuote: true
```

Comment:
`singleQuote` here can be removed. Then the single quotes in `action.yml` will be replaced by double quotes.